src/js/media-controller.js

```diff
@@ -48,6 +48,7 @@ class MediaController extends MediaContainer {
   }
 
   #hotKeys = new AttributeTokenList(this, 'hotkeys');
+  #fullscreenElement;
```

Comment:
I have a little concern about the naming here. will fullscreen be the only feature that needs a settable element?

Reply:
yes, came back to me. the resize observer will also have to be set to this element right?

Reply:
@luwes Yeah I was struggling with a name here, too (names are hard). I definitely don't feel confident that there isn't a future world where there isn't either:
1. a single prop/attr that treats many things as the "media container substitute" (we've discussed some of these in the past, e.g. gestures and keypresses)
2. multiple props that are for setting elements for different purposes on a `media-controller`

Because of that, I didn't want to be overly presumptuous about the name right now (since, at least for case 1, that would be a breaking change). That said, I also don't like the specific name, so if you have an alternative proposal that doesn't assume too much of what it _**could**_ be used for in the future, I'm down.

Reply:
`#fullscreenElement` seemed okay to me since this is a narrow use case, but if we're looking for something that would also apply to the resize observer `#containerElement`?

Then again, there could be use cases where you want the fullscreenElement to be one thing, and some other element to be another thing and not be forced to make them the same thing. For that reason, I'd go with `#fullscreenElement`

Reply:
true, it's hard to say at the moment. if the rumors are true we'll have at least pipElement soon too.

it's not it will be a breaking change I think. 

we could add containerElement later as well and make it clear that it's the element of the full application in any presentation mode.